File: BasicTraining/Web Technologies/jQuery/employee.js
class Employee{
    constructor(id, name, address, salary){
        this.empId = id;
        this.empName = name;
        this.empSalary = salary;
        this.empAddress = address;
    } 
}

class EmployeeRepo{
    constructor(){
        this.data = [];
    }

    addNewEmployee = (emp) => this.data.push(emp);

    deleteEmployee = (id) => {
        const index = this.data.findIndex((rec)=> rec.empId == id);
        this.data.splice(index, 1);
    }

    updateEmployee = (emp) => {
        const index = this.data.findIndex((rec)=> rec.empId == emp.empId);
        this.data.splice(index, 1, emp);
    }
    getAllEmployees = () => this.data;

    findEmployee = (id) => this.data.find((e)=>e.empId == id);
}

File: AdvancedTraining/Proj2-SampleMvcApp/Models/Employee.cs
namespace SampleMvcApp.Models
{
    public class Employee
    {
        public int EmpId{ get; set; }
        public string Name { get; set; } = string.Empty;
        public string Address { get; set; } = string.Empty;
        public int Salary { get; set; }
        public long MobileNo { get; set; }
    }
}
